app/tree/index.html

```diff
@@ -1,6 +1,14 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <meta http-equiv="refresh" content="0; url=/classic/tree" />
-</head>
+  <head>
+    <script>
+      // redirect to /lab by default
+      (function() {
+        window.location.href = window.location.href.replace(
+          /\/(index.html)?$/,
```

Comment:
We might also want to catch the case when coming from the "Launch Classic Notebook" menu entry in lab:

![image](https://user-images.githubusercontent.com/591645/115087254-1d9b2f00-9f0e-11eb-91ae-034d366e61f3.png)

Otherwise it looks like it's going into an infinite redirect loop when hitting `/tree`.

Reply:
bde8c6d

Reply:
Thanks!